### PR TITLE
fix robot deletion event

### PR DESCRIPTION
**Pull request by wy65701436**
Thank you for contributing to Harbor!

# Comprehensive Summary of your change

# Issue being fixed
Fixes #(issue)

Please indicate you've done the following:
- [x] Well Written Title and Summary of the PR
- [x] Label the PR as needed. "release-note/ignore-for-release, release-note/new-feature, release-note/update, release-note/enhancement, release-note/community, release-note/breaking-change, release-note/docs, release-note/infra, release-note/deprecation"
- [x] Accepted the DCO. Commits without the DCO will delay acceptance.
- [x] Made sure tests are passing and test coverage is added if needed.
- [ ] Considered the docs impact and opened a new docs issue or PR with docs changes if needed in [website repository](https://github.com/goharbor/website).

### CODE REVIEW

**Comment by OrlinVasilev** [comment]
@wy65701436 is that WIP?

File: src/controller/scan/callback.go [hunk 1]
```diff
@@ -92,7 +92,7 @@ func scanTaskStatusChange(ctx context.Context, taskID int64, status string) (err
 
 		robotID := getRobotID(t.ExtraAttrs)
 		if robotID > 0 {
-			if err := robotCtl.Delete(ctx, robotID); err != nil {
+			if err := robotCtl.Delete(ctx, robotID, &robot.Option{Operator: "job-service"}); err != nil {
```
Comment (chlins) [inline src/controller/scan/callback.go:95]: should we use `harbor-jobservice` to keep uniformity? https://github.com/goharbor/harbor/blob/969384cd6340f67ceb2d811309c23ec0b1e343ad/src/common/secret/store.go#L19-L20

Comment (wy65701436) [reply src/controller/scan/callback.go]: updated

**Review by chlins — APPROVED** [review]
lgtm

**Review by stonezdj — APPROVED** [review]

**Pull request merged**